### ISSUE TITLE
Upgrade to latest Keycloak (10.0.2)

### ISSUE THREAD

**Issue by gjvoosten** [ahus1/keycloak-dropwizard-integration#31]

**Comment by ahus1** [comment]
Thank you for your three tickets and three pull requests. They are all merged now. It is good to see all dependencies to be up-to-date again and that the tests are still green. 

Any more changes needed on your side before I upgrade that? You can test the 1.1.2-SNAPSHOT version from the Sonatype Snapshot repository. See the main README.adoc for more information.

Again, thank you very much for the updates. Let me know if this is a good time for a release.

**Comment by gjvoosten** [comment]
I'll need to check. With 1.1.1 (so without my latest changes; thanks for merging BTW!) I am getting errors like:
```
[2020-05-14 13:27:17.681] ERROR org.keycloak.adapters.NodesRegistrationManagement: failed to register node to keycloak
[2020-05-14 13:27:17.682] ERROR org.keycloak.adapters.NodesRegistrationManagement: status from server: 200
[2020-05-14 13:27:17.682] ERROR org.keycloak.adapters.NodesRegistrationManagement:    {"error":"unknown_error"}
```
in my application and corresponding errors like:
```
13:27:17,695 WARN  [org.keycloak.events] (default task-3) type=REGISTER_NODE_ERROR, realmId=ANET Realm, clientId=ANET-Client, userId=null, ipAddress=127.0.0.1, error=invalid_client, client_auth_method=client-secret
```
in Keycloak (even though the application works), and it isn't clear to me whether this is due to some misconfiguration on my side, an incompatibility with Keycloak v10 or perhaps code in keycloak-dropwizard that needs to be updated.

**Comment by ahus1** [comment]
The error on the client side is logged here: https://github.com/keycloak/keycloak/blob/master/adapters/oidc/adapter-core/src/main/java/org/keycloak/adapters/NodesRegistrationManagement.java

What version of the Keycloak server are you using?

**Comment by gjvoosten** [comment]
Hmm, for some reason this didn't get posted yesterday. Trying again:

Just checked: the errors were because I didn't have service accounts enabled for my client.
Using 1.1.2-SNAPSHOT from latest master works for me. The are some deprecation warnings in the code (e.g. related to `JaxrsBearerTokenFilterImpl`) and I needed to add a dependency:
```
        <dependency>
            <groupId>org.glassfish.jersey.inject</groupId>
            <artifactId>jersey-hk2</artifactId>
            <version>2.30.1</version>
        </dependency>
```
to be able to run e.g. `mvn test -pl keycloak-dropwizard-jaxrs-example -am -Pkeycloak-dropwizard-jaxrs-example` but otherwise it seems OK.

I'm running Keycloak 10.0.1 BTW. Sorry for the confusion.

**Comment by gjvoosten** [comment]
@ahus1 Keycloak 10.0.2 works fine too, so maybe you want to do another minor upgrade before cutting a 1.1.2 release (which I'd welcome).

**Comment by ahus1** [comment]
Thanks for the reminder. 

I published 1.1.2 with a Keyloak 10.0.2 dependency. It is already available on maven central.
